index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome To Ayodeji's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+    <section class="header_headline_section">
+        <header>
+        <nav>
+            <div class="header_logo"><a href="" class="logo">My Logo</a></div>
+            <div class="header_icon"><img src="icons/Icon-Menu.png" alt="" srcset=""></div>
+        </nav>
+    </header>
+    <section class="headline">
+        <p class="intro_text">I’m Ayodeji. Glad to see you!</p>
+        <p class="intro_summary">
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <p class="connect_social">LET’S CONNECT</p>
+        <div class="social_icon_links">
+            <a href=""><img src="icons/twitter.svg" alt="" srcset=""></a>
+            <a href=""><img src="icons/linkedin.svg" alt="" srcset=""></a>
+            <a href=""><img src="icons/m.svg" alt="" srcset=""></a>
+            <a href=""><img src="icons/git.svg" alt="" srcset=""></a>
+            <a href=""><img src="icons/two.svg" alt="" srcset=""></a>
+        </div>
```

Comment:
- [ ] The list of icons should be a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.